docs/manual-CN/get-started.md

```diff
@@ -133,7 +112,7 @@ nebula 遵循 c++14 标准，依赖第三方库：
 
 * 配置 nebula-metad.conf
 
-   在 nebula 安装目录下，运行以下命令：
+   在 nebula 安装目录(/usr/local/nebula/)下，运行以下命令：
```

Comment:
this is the default install path

Reply:
I just want to make it clear here, thanks for your suggestion.